code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -474,26 +475,32 @@ var/global/image/plasmaman_on_fire = image("icon"='icons/mob/OnFire.dmi', "icon_
 	safe_toxins_max = 0
 	dangerous_existence = 1 //So so much
 	need_nutrition = 0 //Hard to eat through a helmet
+	burnmod = 2
+	heatmod = 2
+	speedmod = 1
 	var/skin = 0
 
 /datum/species/plasmaman/skin
 	name = "Skinbone"
 	skin = 1
+	roundstart = 0
 
 /datum/species/plasmaman/update_base_icon_state(mob/living/carbon/human/H)
 	var/base = ..()
-	if(base == id)
-		base = "[base][skin]"
+	if(base == id && !skin)
+		base = "[base]"
+	else
+		base = "skinbone_m_s"
 	return base
 
 /datum/species/plasmaman/spec_life(mob/living/carbon/human/H)
 	var/datum/gas_mixture/environment = H.loc.return_air()
 
-	if(!istype(H.w_uniform, /obj/item/clothing/under/plasmaman) || !istype(H.head, /obj/item/clothing/head/helmet/plasmaman))
+	if(!istype(H.w_uniform, /obj/item/clothing/under/plasmaman) || !istype(H.head, /obj/item/clothing/head/helmet/space/plasmaman))
 		if(environment)
 			var/total_moles = environment.total_moles()
 			if(total_moles)
-				if((environment.oxygen /total_moles) >= 0.01)
+				if((environment.oxygen /total_moles) >= 1) //Slightly raising this because there can be trace low amounts of oxygen even when you completely vacuum a room.
 					H.adjust_fire_stacks(0.5)
```

Comment:
oh, yeah that was my bad, i just sorta tossed that in quickly to deal with an issue, and yeah i just noticed the invisible man thing. I swear to god I'm never using sprites again.